### PR TITLE
Implement exercise rotational-cipher

### PR DESCRIPTION
**Pull request by MattH-be**

### CODE REVIEW

File: exercises/rotational-cipher/rotational-cipher.spec.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+describe('Rotational cipher', () => {
+  test('rotate a by 0, same output as input', () => {
+    const expected = 'a';
+    const actual = RotationalCipher.rotate('a', 0);
+
+    expect(actual).toEqual(expected);
+  });
+
+  test('rotate a by 1', () => {
```
Comment (matthewmorgan) [inline exercises/rotational-cipher/rotational-cipher.spec.js]: We want to skip all tests but the first, right?

File: exercises/rotational-cipher/rotational-cipher.spec.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    expect(actual).toEqual(expected);
+  });
+
+  test('rotate a by 26, same output as input', () => {
+    const expected = 'a';
+    const actual = RotationalCipher.rotate('a', 26);
+
+    expect(actual).toEqual(expected);
+  });
+
```
Comment (matthewmorgan) [inline exercises/rotational-cipher/rotational-cipher.spec.js]: Looks like there is one test missing, belonging here, that is defined the [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/rotational-cipher/canonical-data.json#L32):

```json
        {
          "description": "rotate m by 13",
          "property": "rotate",
          "text": "m",
          "shiftKey": 13,
          "expected": "z"
        },
```

Comment (MattH-be) [reply exercises/rotational-cipher/rotational-cipher.spec.js]: Correct! Must have missed that one, should be fixed now.

**Comment by MattH-be** [comment]
Everything should be ready to merge now. Had an issue where a test failed because I replaced all test with 'xtest' but a test input contained the string test.

**Comment by matthewmorgan** [comment]
Thanks @MattH-be !!

**Pull request merged**